RestCommClient/Classes/RCCallKitProvider.h

```diff
@@ -38,24 +37,31 @@
 - (void)callEnded;
 @end
 
+/**
+ *  @abstract RCCallKitProvider handles the CallKit logic.
+ *  @param connection The RCConnection instance
+ */
 @interface RCCallKitProvider : NSObject <CXProviderDelegate, RCConnectionDelegate>
 
 @property (nonatomic, strong) RCConnection * connection;
-@property (nonatomic, strong) NSUUID *currentUdid;
 
-- (id)initWithDelegate:(id<RCCallKitProviderDelegate>)delegate;
-
-- (void)initRCConnection:(RCConnection *)connection;
+/**
+ *  @abstract Initialize a new instance
+ *  @param delegate The RCCallKitProviderDelegate delegate instance
+ *  @param imageName Resource's image name which will be shown on callkit view 
```

Comment:
Are there any specs for the image to be passed here, like dimensions, etc? If so we should document it

Reply:
Yes they are I have put the description.